test/widgets/settings_test.dart

```diff
@@ -160,6 +160,24 @@ void main() {
     }, variant: TargetPlatformVariant({TargetPlatform.android, TargetPlatform.iOS}));
   });
 
+  group('SettingPageScrollBehavior', () {
```

Comment:
This string looks _like_ an identifier, like the name of a class or something, but then I don't find anything when I look for its definition. To be less confusing, we should just say "SettingsPage scroll behavior" or similar. (`SettingsPage` is the name of a class, so it doesn't add confusion to use that name.)